internal/cloudevents/payload.go

```diff
@@ -0,0 +1,28 @@
+package cloudevents
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I think the resource definition should be in the api repo finally?

Reply:
@skeeey will create a directory in api repo holds the previously mentioned manifest and associated payload, as per our earlier conversation.